src/data/apiCalls.js

```diff
@@ -7,7 +7,7 @@ const apiRequest = path => {
       throw new Error('Error');
     }
   })
-  .catch(error => console.log(`Could not fetch because: ${error}`))
+  .catch(error => console.log(`Could not fetch because: ${error}`));
```

Comment:
Probably a good idea to move this catch into the app.js - so we can do something with that error message that would benefit the user!